packages/moonstone/Scrollable/ScrollableNative.js

```diff
@@ -95,11 +97,51 @@ class ScrollableNative extends Component {
 		 * @default false
 		 * @public
 		 */
-		focusableScrollbar: PropTypes.bool
+		focusableScrollbar: PropTypes.bool,
+
+		/**
+		* Sets the hint string read when focusing the next button in the horizontal scroll bar.
+		*
+		* @type {String}
+		* @default $L('scroll right')
+		* @public
+		*/
+		horizontalScrollbarNextButtonAriaLabel: PropTypes.string,
+
+		/**
+		* Sets the hint string read when focusing the previous button in the horizontal scroll bar.
+		*
+		* @type {String}
+		* @default $L('scroll left')
+		* @public
+		*/
+		horizontalScrollbarPreviousButtonAriaLabel: PropTypes.string,
+
+		/**
+		* Sets the hint string read when focusing the next button in the vertical scroll bar.
+		*
+		* @type {String}
+		* @default $L('scroll down')
+		* @public
+		*/
+		verticalScrollbarNextButtonAriaLabel: PropTypes.string,
+
+		/**
+		* Sets the hint string read when focusing the previous button in the vertical scroll bar.
+		*
+		* @type {String}
+		* @default $L('scroll up')
+		* @public
+		*/
+		verticalScrollbarPreviousButtonAriaLabel: PropTypes.string
 	}
 
 	static defaultProps = {
-		focusableScrollbar: false
+		focusableScrollbar: false,
+		horizontalScrollbarNextButtonAriaLabel: $L('scroll right'),
```

Comment:
`$L` shouldn't be used in `defaultProps` because it will only be evaluated at component definition time and not reflect changes to the locale. For now, I recommend include a null check inline in render. When async ilib lands, we'll likely have a different solution but that is yet to be defined.

```
nextButtonAriaLabel={horizontalScrollbarNextButtonAriaLabel == null ? $L('scroll right') : horizontalScrollbarNextButtonAriaLabel}
```